README.md

```diff
@@ -34,22 +34,7 @@ end
 
 ### Mission Editing
 
-Add the following code to your mission. This will start the DCS-gRPC server. You can add this code to a DOSCRIPT
-trigger in the mission editor or include it in your own scripts if you already have some.
-
-```lua
-package.cpath = package.cpath..lfs.writedir()..[[Mods\tech\DCS-gRPC\?.dll;]]
-GRPC = { basePath = lfs.writedir()..[[Scripts\DCS-gRPC\]] }
-
-local luaPath = GRPC.basePath .. [[grpc.lua]]
-local f = assert( loadfile(luaPath) )
-
-if f == nil then
-  error ("[GRPC]: Could not load " .. luaPath )
-else
-  f()
-end
-```
+Create a trigger of type `MISSION START` and add a `DO SCRIPT FILE` action loading `Scripts\DCS-gRPC\grpc-mission.lua`.
```

Comment:
I thought it might be easier to just load a Lua file, wdyt? I'd also be fine with changing it back.

Reply:
We need something that is fail-safe. Not everyone who works on a mission will have DCS-gRPC installed locally. (e.g. people modifying and testing a Hoggit mission where gRPC export is not needed so we cannot assume that `grpc-mission.lua` is present.

I think for maximum nicety to mission devs I would go with the above `DO SCRIPT` instead of `DO SCRIPT FILE`

Reply:
Kept both 👍